nipype/interfaces/fsl/model.py

```diff
@@ -1134,28 +1134,28 @@ class L2Model(BaseInterface):
 
     def _run_interface(self, runtime):
         cwd = os.getcwd()
-        mat_txt = ['/NumWaves       1',
-                   '/NumPoints      %d' % self.inputs.num_copes,
-                   '/PPheights      %e' % 1,
+        mat_txt = ['/NumWaves   1',
+                   '/NumPoints  {}'.format(self.inputs.num_copes),
+                   '/PPheights  1',
```

Comment:
`'%e' % 1` creates `'1.000000e+00'`, not `1`. I'm not sure what happens if you don't pass this as a float in FSL, but I think if you're just making cosmetic changes you should take all efforts to avoid functional changes.